docs/bigquery/snippets.py

```diff
@@ -2579,6 +2579,178 @@ def test_query_no_cache(client):
     # [END bigquery_query_no_cache]
 
 
+def test_query_external_gcs_temporary_table(client):
+    # [START bigquery_query_external_gcs_temp]
+    # from google.cloud import bigquery
+    # client = bigquery.Client()
+
+    # Configure the external data source and query job
+    external_config = bigquery.ExternalConfig('CSV')
```

Comment:
Maybe use [SourceFormat.CSV](https://google-cloud-python.readthedocs.io/en/latest/bigquery/generated/google.cloud.bigquery.job.SourceFormat.CSV.html#google.cloud.bigquery.job.SourceFormat.CSV) instead of a plain string?

Reply:
You're right that we should probably create a separate set of constants since the set of supported formats aren't the same. For example, you can't use Parquet in an external config yet.

Reply:
Created issue https://github.com/GoogleCloudPlatform/google-cloud-python/issues/5499